superset/assets/visualizations/filter_box.jsx

```diff
@@ -10,19 +10,25 @@ import { TIME_CHOICES } from './constants';
 import './filter_box.css';
 
 const propTypes = {
-  origSelectedValues: PropTypes.object,
-  instantFiltering: PropTypes.bool,
   filtersChoices: PropTypes.object,
   onChange: PropTypes.func,
   showDateFilter: PropTypes.bool,
   datasource: PropTypes.object.isRequired,
+  showLabel: PropTypes.bool,
+  showMetricNumber: PropTypes.bool,
+  origSelectedValues: PropTypes.object,
+  instantFiltering: PropTypes.bool,
+  sliceId: PropTypes.number,
```

Comment:
Are we actually using sliceId for something?

Reply:
It appears not. That was in there when I began work on the filter_box, so I added to prop_types for consistency...Let me know and I'll remove it from the entire file.

Reply:
If sliceId is not used no need to keep it.

Reply:
I will remove that now